pkg/listwatch/listwatch_test.go

```diff
@@ -456,7 +456,7 @@ func TestMultiWatchDoesNotDeadlock(t *testing.T) {
 			},
 			WatchFunc: func(options metav1.ListOptions) (watch.Interface, error) {
 				if namespace == "4" {
-					return nil, errors.New("Some error occurred")
+					return nil, errors.New("some error occurred")
```

Comment:
is there a reason we don't return the error? Also, would it be better to say "an error occurred"?

Reply:
If this is not code we have authored then we should keep it matching upstream, the only possible link error would be the capitalisation.

Reply:
oh. Just realized this is in test code anyway. But I think we added the listwatcher for the multitenancy work didn't we?

Reply:
No, they had already copied a version of it from the prometheus codebase